src/main/index.ts

```diff
@@ -65,15 +65,18 @@ log.transports.file.format = "[{y}-{m}-{d} {h}:{i}:{s}.{ms}][{processType}][{lev
 log.eventLogger.format = "Electron event {eventSource}#{eventName} observed";
 
 log.hooks.push((message, transport) => {
-  if (transport !== log.transports.file) return message;
-
-  // Annoyingly spammed log from YTM that doesn't need to get added to the log file
-  if (message.data[0].includes("Third-party cookie will be blocked.")) return false;
-  for (let i = 0; i < message.data.length; i++) {
-    let dataString: string = message.data[i] as string;
-    dataString = dataString.replaceAll(/(?<=((https|http):\/\/)?.{1,64}(\..{1,64})?\..{1,64}\/)([\S]+)/gm, "[REDACTED]");
-    message.data[i] = dataString;
-  }
+  // If the transport is not a file transport then return as is
+  if (transport !== log.transports.file) { return message; }
+  // If there isnt message data, or the data isnt a string, or the data is spam from Youtube Music, return false
+  if (message?.data?.[0] && (typeof message.data[0] === 'string' && message.data[0].includes("Third-party cookie will be blocked."))) return false;
+
+  // Check it is an array, then redact sensitive info
+  message.data = message.data.map(data => {
+    if (typeof data === 'string') {
+      return data.replaceAll(/(?<=((https|http):\/\/)?.{1,64}(\..{1,64})?\..{1,64}\/)([\S]+)/gm, "[REDACTED]");
+    }
+    return data;
+  });
```

Comment:
As mentioned, please do not Cross your PRs.
1 PR = 1 Change of stuff.

You already have #1400 for this changes, and we will approve them seperately.

Reply:
Legit cannot console.log anything

Reply:
?
As mentioned, don't mix your PRs. You already have submitted this code separately inside it #1400 . We don't need it to be committed twice.